src/cgame/cg_main.cpp

```diff
@@ -1409,6 +1394,11 @@ void CG_Init( int serverMessageNum, int clientNum, const glconfig_t& gl, const G
 	new(&cg) cg_t{};
 	memset( cg_entities, 0, sizeof( cg_entities ) );
 
+	cg.loadingFraction = 0.0f;
```

Comment:
This is redundant since they were just cleared with `new(&cg) cg_t{}`

Reply:
Right…